yfinance/exceptions.py

```diff
@@ -1,12 +1,49 @@
 class YFinanceException(Exception):
```

Comment:
Can we shorten all instances of "YFinance" in exception class names to "YF"?
Can make other shortenings eg:
- timezone -> TZ
- priceData -> prices